install_config/configuring_nsxtsdn.adoc

```diff
@@ -0,0 +1,239 @@
+
+This example topology assumes you are using 3 {product-title} master virtual machines and 4 {product-title} worker virtual machines (two for infrastructure and two for compute).
+
+[[nsx-t-sdn-installation]]
+== Installing vmWare NSX-T 
+
+Prerequisites:
+
+* ESXi hosts requirements:
+** ESXi servers that host {product-title} node VMs must be NSX-T Transport Nodes.
++
+.NSX UI dislaying the Transport Nodes for a typical high availability environment: 
++
+image::nsxt-transportnodes.png[NSX Transport Nodes]
+
+* DNS requirements:
+** You must add a new entry to your DNS server with a wildcard to the infrastructure nodes. This allows load balancing by NSX-T or other third-party LoadBalancer. In the `hosts` file below, the entry is defined by the `openshift_master_default_subdomain` variable.
+** You must update your DNS server with the `openshift_master_cluster_hostname` & `openshift_master_cluster_public_hostname` variables.
+
+* Virtual Machine requirements:
+** The {product-title} node VMs must have two vNICs:
+** A Management vNIC must be connected to the Logical Switch that is uplinked to the management T1 router.
+** The second vNIC on all VMs must be tagged in NSX-T so that the NSX Container Plugin (NCP) knows which port needs to be used as a parent VIF for all Pods running in a particular {product-title} node. The tags must be the following:
++
+----
+{'ncp/node_name': 'node_name'}
+{'ncp/cluster': 'cluster_name'}
+----
++
+The following image shows how the tags in NSX UI for all nodes. For a large scale cluster, you can automate the tagging using API Call or by using Ansible. 
++
+.NSX UI dislaying node tags
++
+image::nsxt-tags.png[NSX VM tags]
++
+The order of the tags in the NSX UI is opposite from the API.
+The node name must be exactly as kubelet expects and the cluster name must be the same as the `nsx_openshift_cluster_name` in the Ansible hosts file, as shown below. Ensure that the proper tags are applied on the second vNIC on every node.
++
+* NSX-T requirements: 
++
+The following prerequisites need to be met in NSX:
++
+** A Tier-0 Router.
+** An Overlay Transport Zone.
+** An IP Block for POD networking.
+** Optionally, an IP Block for routed (NoNAT) POD networking.
+** An IP Pool for SNAT. By default the subnet given per Project from the Pod networking IP Block is routable only inside NSX-T. NCP uses this IP Pool to provide connectivity to the outside.
+** Optionally, the Top and Bottom firewall sections in a dFW (Distributed Firewall). NCP places the Kubernetes Network Policy rules between those two sections.
+** The Open vSwitch and CNI plugin RPMs need to be hosted on a HTTP server reachable from the {product-title} Node VMs (`http://websrv.example.com` in this example). Those files are included in NCP Tar file, which you can download from vmWare at link:https://my.vmware.com/web/vmware/details?downloadGroup=NSX-T-PKS-240&productId=673[Download NSX Container Plugin 2.4.0
```

Comment:
s/plugin/plug-in
s/in NCP Tar file/in the NCP tar file
s/vmWare/VMware

Reply:
Did they change vmWare? Could sworn it was that way, but you are right....